src/library_sdl.js

```diff
@@ -1846,10 +1846,14 @@ var LibrarySDL = {
   SDL_SetError: function() {},
 
   SDL_malloc__sig: 'ii',
-  SDL_malloc: 'malloc',
```

Comment:
How did this work previously?

Reply:
It worked thanks to how aliases work: `a: "b"` means `a` is an alias of `b`, so it just emits
```javascript
function a() { b() }
```
Basically it does an indirection. And that happens to work if `b` is in JS or in compiled code. It is confusing though which is why I changed it.

Thinking on it now, though, even better would be to implement these in C. However it's for SDL1 and not 2 so it matters less.